pkg/async/notifications/factory.go

```diff
@@ -41,6 +43,17 @@ type EmailerConfig struct {
 }
 
 func GetEmailer(config runtimeInterfaces.NotificationsConfig, scope promutils.Scope) interfaces.Emailer {
+	// If an external email service is specified use that first.
+
+	if config.NotificationsEmailerConfig.EmailerConfig.ServiceName != ""{
```

Comment:
might be worth refactoring to to make the EmailerConfig accept 'aws'  and 'local' as a service name so we don't have to do two types of checks

Reply:
I can add that, but I don't want to remove the existing check - if I do, it'll be a breaking config change.  or you think it's worth it?

Reply:
sure sure let's avoid a breaking change in this PR. maybe we can file an issue to track cleaning up the emailer configs

Reply:
https://github.com/flyteorg/flyte/issues/1063